sdk/src/main/java/com/mapbox/maps/ViewAnnotationManagerImpl.kt

```diff
@@ -68,8 +68,6 @@ internal class ViewAnnotationManagerImpl(
 
   override fun removeViewAnnotation(view: View): Boolean {
     val id = idLookupMap.remove(view) ?: return false
-    val annotation = annotationMap.remove(id) ?: return false
```

Comment:
So we don't need to remove the annotation from annotationMap?

Reply:
good catch - we actually do 😄 
thanks, will fix!